pkg/lvm/lvm_util.go

```diff
@@ -86,7 +86,33 @@ func buildLVMCreateArgs(vol *apis.LVMVolume) []string {
 	}
 
 	LVMVolArg = append(LVMVolArg, vol.Spec.VolGroup)
+	return LVMVolArg
+}
+
+// builldThinLVMCreateArgs returns lvcreate command for thin volume
+func buildThinLVMCreateArgs(vol *apis.LVMVolume) []string {
```

Comment:
we can merge this with buildLVMCreateArgs itself?

Reply:
i did this earlier in single func, which is kind of confusing with so many if conditions checks, so thought of making it clear using different func

Reply:
@prateekpandey14 we should not create a new function for each argument. It seems likewe need one if condition for thin provisioning to do all the related task there?